package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "finbars",
-  "version": "1.5.2",
-  "main": "index.js",
+  "version": "1.5.3",
+  "main": "dist/common.js",
```

Comment:
Not really sure how you were publishing the code, and because we needed to be able to install from a branch, I've moved `src/index.js` to `dist/common.js` and add it to git.